src/i18n/locales/ko-KR/settings.json

```
@@ -51,6 +52,7 @@
 		"blurBg": "배경 흐림",
 		"motionBlur": "모션 블러",
 		"off": "끄기",
+		"on": "켜기",
```

Biome (2.4.15)

[error] The key on was already declared.

(lint/suspicious/noDuplicateObjectKeys)